Command/DoctrineCommand.php

```
@@ -52,36 +46,13 @@ protected function getEntityGenerator()
     /**
      * Get a doctrine entity manager by symfony name.
      *
-     * @param string   $name
-     * @param int|null $shardId
+     * @param string $name
      *
      * @return EntityManager
      */
-    protected function getEntityManager($name, $shardId = null)
+    protected function getEntityManager($name)
```

Passing a shard ID to this method should raise an exception as before.